scripts/jvmrun

```diff
@@ -0,0 +1 @@
+-heapsize 25M -classpath /usr/flexy-thingworx-connector.jar -emain com.hms_networks.americas.sc.thingworx.TWConnectorMain
```

Comment:
Was 25M found through testing or just following the same as other connectors? We would want to claim as little heap as possible.

Reply:
It has stuck from previous connectors but has also been tested to be the most stable/optimal setting.